pset-iterators.js

```diff
@@ -115,6 +141,14 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const presidentsBetween1850and1900 = presidents.filter(function (name) {
```

Comment:
@jacobkelman nice work, `const` would have been more applicable instead of `let` when declaring the `splits` variable since you won't have a need to reassign a  new value to `splits` after the initial declaration